MIGRATION.md

```diff
@@ -0,0 +1,16 @@
+#Migrate AWS RDS databases to use different private subnets
+
+This procedure explains how to migrate AWS RDS databases, deployed with ref architecture, from using separate, private DB subnets to (re-)using 
+private subnets already deployed with VPC. This only applies to already existing deployments, new deployments are not affected.
+The change was introduced with v0.2.0 of reference architecture.
+
+##Steps:
+1. Check your existing deployment and make sure everything works etc. (no issues, pods not starting and so on)
+2. Delete keycloak and simphera databases, making sure that final snapshot is created
+3. After DBs are deleted, update DB subnet group, remove DB specific subnets and add VPC private subnets
+4. Restore DBs from snapshots taken, selecting same configuration options as original DBs (names, VPCs, security groups etc.)
+5. Remove old DB subnet group from TF state, eg.: terraform state rm module.vpc.aws_db_subnet_group.database[0]
+6. Import new DB subnet group into TF state, eg.: terraform import module.simphera_instance[\"production\"].aws_db_subnet_group.default <infrastructure name>-vpc
+7. Re-run TF to refresh the state and apply any small configuration changes in-place, which were missed in previous steps
```

Comment:
Suggestion:
7. Let Terraform apply the remaining changes and refresh the Terraform state by re-running `terraform apply`

Reply:
Done